src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
+            // Форматтер для работы с датами
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
If you create a formatter, make it a constant field. That will make your code easier to understand and more maintainable.